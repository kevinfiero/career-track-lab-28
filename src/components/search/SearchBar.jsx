import React from 'react';
import PropTypes from 'prop-types';

const SearchBar = () => (
  <>
    <input type="text"></input>
  </>


);

export default SearchBar;
