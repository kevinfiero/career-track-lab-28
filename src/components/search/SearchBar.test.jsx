import React from 'react';
import { render, cleanup } from '@testing-library/react';
import SearchBar from './SearchBar';

describe('SearchBar component', () => {
  afterEach(() => cleanup());
  it('renders SearchBar', () => {
    const { asFragment } = render(<SearchBar 
      search = "Tesla"
      onChange = {jest.fn()}
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
