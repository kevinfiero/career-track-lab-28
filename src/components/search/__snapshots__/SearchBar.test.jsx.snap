// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component renders SearchBar 1`] = `
<DocumentFragment>
  <label
    for="search"
  >
    Search Here:
  </label>
  <input
    id="search"
    name="search"
    type="text"
    value="Tesla"
  />
</DocumentFragment>
`;
